test/exercism/comment_test.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../integration_helper'
+require 'minitest/pride'
+
+class CommentTest < Minitest::Test
+  include DBCleaner
+
+  attr_reader :alice, :fred, :comment, :submission
+  def setup
+    super
+    @alice = User.create(username: 'alice')
+    @fred = User.create(username: 'fred')
+    @submission = Submission.create(user: alice, language: 'python', slug: 'one')
+    # @comment2 = Comment.create(user: fred, submission: sub1, body: 'something')
+  end
+
+  def test_is_qualifying
+    submission.reload
+    comment = Comment.create(user: fred, submission: submission, body: 'something')
+    assert_equal true, comment.qualifying?
+  end
+
+  def test_is_not_qualifying
+    submission.reload
+    comment = Comment.create(user: alice, submission: submission, body: 'something')
+    assert_equal false, comment.qualifying?
+  end
+
+  def test_qualifying_comment_adds_to_table
```

Comment:
We don't have to test both `increment_five_a_day` together with the `if comment.qualifying?`.